src/MultiversX/Serialization.cpp

```diff
@@ -23,8 +23,9 @@ std::map<std::string, int> fields_order{
     {"data", 9},
     {"chainID", 10},
     {"version", 11},
-    {"options", 12},
-    {"signature", 13}};
+    {"signature", 12},
+    {"options", 13},
```

Comment:
The "options" field wasn't previously used - if it were, it would have led to an incorrect serialization, and the transaction would have been rejected by the Network. This PR corrects the serialization around "options" - bullet 2 of the description: 

> The Serialization logic has been fixed to correctly include the options (and the new guardian field).

Thus, in our understanding, this shouldn't be considered a breaking change. Please correct me if mistaken.